src/dev/license_checker/config.ts

```diff
@@ -67,6 +67,7 @@ export const LICENSE_WHITELIST = [
   'Unlicense',
   'WTFPL OR ISC',
   'WTFPL',
+  'Nuclide software',
```

Comment:
@peterschretlen is this 👍?

Reply:
@spalger thanks for flagging - I had a look at the [license](https://github.com/facebook-atom/nuclide-prebuilt-libs/blob/master/LICENSE). I suspect its fine but we'll need to have it reviewed/approved if it's not already whitelisted. I'll get that started.

Reply:
Review is done - this addition to the whitelist is approved.